### PR TITLE
Add Watcher Test Case

### PR DESCRIPTION
**Pull request by freyes**
This PR introduces a new test case for OpenStack Watcher which uses the vm_workload_consolidation strategy[0]

Used/verified by this gerrit review:
https://review.opendev.org/c/openstack/charm-watcher/+/891195

[0] https://docs.openstack.org/watcher/latest/strategies/basic-server-consolidation.html

### CODE REVIEW

File: zaza/openstack/configure/guest.py [hunk 1]
```diff
@@ -125,13 +127,20 @@ def launch_instance(instance_key, use_boot_volume=False, vm_name=None,
     :type keystone_session: Optional[keystoneauth1.session.Session]
     :param perform_connectivity_check: Whether to perform a connectivity check.
     :type perform_connectivity_check: bool
+    :param host: Requested host to create servers
+    :type host: str
+    :param nova_api_version: Nova API version to use
+    :type nova_api_version: str
```
Comment (ajkavanagh) [inline zaza/openstack/configure/guest.py:133]: I think this should be:

    :type nova_api_version: str | None

**Review by ajkavanagh — COMMENTED** [review]
LGTM; one minor  comment on a docstring.

**Comment by ajkavanagh** [comment]
@freyes just pending a gerrit review that exercises this; please could you link one into the description for posterity' sake?

**Comment by freyes** [comment]
this PR is being exercised by this gerrit review:
https://review.opendev.org/c/openstack/charm-watcher/+/891195

File: zaza/openstack/configure/guest.py (hunk 1 above)
Comment (freyes) [reply zaza/openstack/configure/guest.py]: 449164284f297303d67ba61dc6451863b646d027 - this commit addresses this.

**Review by ajkavanagh — APPROVED** [review]

**Pull request merged**